dev/publish_release.sh

```diff
@@ -44,8 +44,9 @@ NC='\033[0m' # No Color
 RELEASE_VERSION="$1"
 GIT_TAG="$2"
 
-PUBLISH_PROFILES="-Pspark-1.6 -Phadoop-2.4 -Pyarn -Ppyspark -Psparkr -Pr"
+PUBLISH_PROFILES="-Pspark-2.0 -Phadoop-2.4 -Pyarn -Ppyspark -Psparkr -Pr"
 PROJECT_OPTIONS="-pl !zeppelin-distribution"
+PROJECT_OPTIONS_SCALA211="-pl zeppelin-interpreter,cassandra,flink,ignite,spark,spark-dependencies,r,zeppelin-display"
```

Comment:
I would recommend though that we do a clean and a full recompile with scala 2.11 to be on the safe mode but
if we are going to build projects individually, we need to include scalding, zeppelin_server and even maybe zeppelin-distribution

grep -r --include=pom.xml "_2.10</" * 
cassandra/pom.xml:    <artifactId>zeppelin-cassandra_2.10</artifactId>
flink/pom.xml:  <artifactId>zeppelin-flink_2.10</artifactId>
ignite/pom.xml:  <artifactId>zeppelin-ignite_2.10</artifactId>
r/pom.xml:  <artifactId>zeppelin-zrinterpreter_2.10</artifactId>
scalding/pom.xml:  <artifactId>zeppelin-scalding_2.10</artifactId>
spark/pom.xml:  <artifactId>zeppelin-spark_2.10</artifactId>
spark-dependencies/pom.xml:  <artifactId>zeppelin-spark-dependencies_2.10</artifactId>
zeppelin-display/pom.xml:  <artifactId>zeppelin-display_2.10</artifactId>
zeppelin-distribution/pom.xml:      <artifactId>zeppelin-server_2.10</artifactId>
zeppelin-server/pom.xml:  <artifactId>zeppelin-server_2.10</artifactId>